src/BloomExe/CollectionTab/LibraryListView.cs

```diff
@@ -856,7 +872,7 @@ bool IsUsableBook(Button bookButton)
 			// Eithe way, the basic idea is that books in the main collection you are now editing are always usable.
 			if (bookButton.Parent == _primaryCollectionFlow)
 				return true;
-			var bookInfo = (BookInfo) bookButton.Tag;
+			var bookInfo = (bookButton.Tag as BookButtonInfo).BookInfo;
```

Comment:
Getting very repetitive...it's probably not worth it now, but I would have been inclined, while changing all these places, to encapsulate in a method the notion of how to get a BookInfo from the tag of a button.